website/docs/integrations/vscode.mdx

```diff
@@ -6,11 +6,12 @@ description: ConfigCat Visual Studio Code extension. This is a step-by-step guid
 
 <a href="https://marketplace.visualstudio.com/items?itemName=ConfigCat.configcat-feature-flags" target="_blank">ConfigCat's Visual Studio Code extension</a> to manage feature flags from Visual Studio Code.
 
-## Feature overview
+Connect your Product and Config to your Workspace. Find your Feature Flag's usages in your code easily.
```

Comment:
Én itt is a "ConfigCat Product and Config" és "Trello Workspace" kifejezéseket javaslom, azért, hogy a nem túl gyakorlatt userek is tudják, hogy melyik dolgog melyik toolban van.